### PR TITLE
BERT Retrain/Large Refactor

### PR DESCRIPTION
**Pull request by ehlong**
Built the bones for retraining to occur for BERT, and did a HUGE refactor to remove shadowing, conform to snake_case, clean up some unused variables, etc.

### CODE REVIEW

File: TweetClassifier/src/Database_Api.py [hunk 1]
```diff
@@ -122,108 +121,105 @@ def update_collection(collection: Collection, new_values: [dict]) -> BulkWriteRe
         if 'class' in document:
             fields['$set']['class'] = document['class']
 
+        if 'trained' in document:
+            fields['$set']['trained'] = document['trained']
         requests.append(pymongo.UpdateOne({'id': document['id']}, fields, upsert=True))
 
-    result = collection.bulk_write(requests, ordered=False)
-    return result
+    validate = db_collection.bulk_write(requests, ordered=False)
```
Comment (jboothby) [inline TweetClassifier/src/Database_Api.py:128]: Why the rename here? The choice of result originally was to reflect that it is a BulkWriteResult object

File: TweetClassifier/src/main.py [hunk 2]
```diff
@@ -112,8 +112,8 @@ def split_data_by_class(train, test, feature_names):
 
 
 def split_data_by_class_cv(x, y, feature_names):
-    x = x
-    y = y
+    # x = x
```
Comment (jboothby) [inline TweetClassifier/src/main.py:115]: We can probably just delete these, I'm not sure what their purpose is

File: TweetClassifier/src/main.py [hunk 3]
```diff
@@ -16,7 +16,7 @@
 from nltk.corpus import stopwords
 
 # TODO: Change preprocess_data_train_test_split for binClass to work
-# allow printing full width in console
```
Comment (jboothby) [inline TweetClassifier/src/main.py:19]: This wasn't part of the todo, it was just a comment explaining the three lines underneath it.

File: TweetClassifier/src/main.py [hunk 4]
```diff
@@ -178,6 +178,54 @@ def train_model_bert():
...
+
+
+
+    db_contents = fetch()
+    ### CHECK ###
+    tweets = [x for x in db_contents if 'trained' not in x.keys()]
+    ### TESTING BELOW ###
+    for tweet in tweets:
+        tweet['trained'] = 1
+    collection = get_database_collection
```
Comment (jboothby) [inline TweetClassifier/src/main.py:197]: Missing parenthesis on the end of that function call

**Review by jboothby — CHANGES_REQUESTED** [review]
See comments below :)

File: TweetClassifier/src/Database_Api.py (hunk 1 above)
Comment (ehlong) [reply TweetClassifier/src/Database_Api.py:128]: Was causing a shadowing issue, resolved in a different way now

File: TweetClassifier/src/main.py (hunk 2 above)
Comment (ehlong) [reply TweetClassifier/src/main.py:115]: Deleted, I thought I deleted them with the original commit -.-

File: TweetClassifier/src/main.py (hunk 3 above)
Comment (ehlong) [reply TweetClassifier/src/main.py:19]: Fixed

File: TweetClassifier/src/main.py (hunk 4 above)
Comment (ehlong) [reply TweetClassifier/src/main.py:197]: (It is (done))

**Review by jboothby — APPROVED** [review]
Looks good to me man

**Pull request merged**